addons/account/models/account_journal_dashboard.py

```diff
@@ -1008,10 +1008,14 @@ def open_action(self):
         action = self.env["ir.actions.act_window"]._for_xml_id(action_name)
 
         context = self._context.copy()
+        include_refund = context.get('include_refund', False)
         if 'context' in action and isinstance(action['context'], str):
             context.update(ast.literal_eval(action['context']))
         else:
             context.update(action.get('context', {}))
+        if include_refund and self.type in ('sale', 'purchase'):
```

Comment:
i think we can put the context.get('include_refund') directly in the if statement, no need to add a variable, also False is the default value of the get when include_refund is not found so no need to put that :smile:

Reply:
> i think we can put the context.get('include_refund') directly in the if statement

We can't, as the context is updated ;-)